Core/Net/AutoUpdate.cs

```diff
@@ -73,8 +79,15 @@ public void FetchLatestReleaseInfo()
 
             try
             {
-                fetchedUpdaterUrl = RetrieveUrl(MakeRequest(latestUpdaterReleaseApiUrl));
-                fetchedCkanUrl = RetrieveUrl(response);
+                fetchedCkanUrl = RetrieveUrl(response, 0);
+                // Check whether the release includes the auto updater
+                if (response.assets.Count >= 2) {
+                    // Second asset is AutoUpdater.exe
```

Comment:
The CKAN.app is now the second asset, and aren't we going to add in a .deb as well? Can we use a regex on the assets to find the correct one, rather than hard-code the position?

Reply:
Yup, this needs to be updated every time we add to that list (which hopefully won't be too many more times). It was 1 before the mac PR got merged, and I'll be incrementing it again now that the deb project is in. Luckily GitHub helpfully notices that these PRs all conflict with one another, which ensures that we don't merge without reconciling them.